polyjuice-tests/src/ctx.rs

```diff
@@ -0,0 +1,575 @@
+use std::{
+    collections::HashMap,
+    convert::TryInto,
+    fs,
+    io::Read,
+    path::{Path, PathBuf},
+    time::SystemTime,
+};
+
+pub use gw_common::builtins::{CKB_SUDT_ACCOUNT_ID, ETH_REGISTRY_ACCOUNT_ID, RESERVED_ACCOUNT_ID};
+use gw_common::{
+    blake2b::new_blake2b,
+    h256_ext::H256Ext,
+    registry_address::RegistryAddress,
+    state::{build_account_key, build_data_hash_key, State},
+    CKB_SUDT_SCRIPT_ARGS, H256,
+};
+use gw_config::{BackendConfig, BackendSwitchConfig, BackendType};
+use gw_db::schema::{COLUMN_INDEX, COLUMN_META, META_TIP_BLOCK_HASH_KEY};
+use gw_generator::{
+    account_lock_manage::{secp256k1::Secp256k1, AccountLockManage},
+    backend_manage::BackendManage,
+    dummy_state::DummyState,
+    traits::StateExt,
+    Generator,
+};
+use gw_store::{
+    chain_view::ChainView,
+    traits::{chain_store::ChainStore, kv_store::KVStoreWrite},
+    Store,
+};
+use gw_traits::CodeStore;
+use gw_types::{
+    bytes::Bytes,
+    core::{AllowedContractType, AllowedEoaType, ScriptHashType},
+    offchain::{RollupContext, RunResult},
+    packed::{AllowedTypeHash, BlockInfo, RawL2Transaction, RollupConfig, Script, Uint64},
+    prelude::*,
+    U256,
+};
+
+use crate::helper::{PolyjuiceArgsBuilder, POLYJUICE_PROGRAM_CODE_HASH};
+pub const CREATOR_ACCOUNT_ID: u32 = 3;
```

Comment:
Why not use the variables defined in helper.rs?

Reply:
Fixed!